setup/vdisk.py

```diff
@@ -345,9 +333,9 @@ def rollback_to_snapshot(vdisk_name, vpool_name, snapshot_id, api, timeout=ROLLB
             VDiskSetup.LOGGER.info("Rollback vDisk `{0}` should have succeeded".format(vdisk_name))
             return task_result[1]
 
-    @staticmethod
+    @classmethod
     @required_vdisk
-    def set_config_params(vdisk_name, vpool_name, config, api, timeout=SET_CONFIG_VDISK_TIMEOUT):
+    def set_config_params(vdisk_name, vpool_name, config, timeout=SET_CONFIG_VDISK_TIMEOUT):
```

Comment:
Missing cls declaration. Snapshot_name is currently treated as the VDiskSetup pointer